src/public/js/util.js

```diff
@@ -257,7 +265,7 @@ var UTIL = (function() {
       }
     }
 
-    return remainder;
+    return trimLeadingZeros(remainder);
```

Comment:
I see that now :P I think trimming there would be fine if we refactor elsewhere. What would you prefer. If we want to stick to the notation of forgetting about the leading 0s or just mix and match.